src/Category/PERM.agda

```diff
@@ -0,0 +1,65 @@
+{-# OPTIONS --type-in-type #-}
+
+module Category.PERM where
```

Comment:
I realized this after we talked, but we could generalize this to work on any arbitrary category (Instead of considering only automorphisms in set, we can consider automorphisms anywhere). Food for thought!

Reply:
I thought of that too! But it's for another day